src/components/mypage/ReservationList/ReservationCard.tsx

```diff
@@ -26,6 +30,24 @@ export default function ReservationCard({
     headCount,
   } = reservation;
 
+  const mutation = useMutation({
+    mutationFn: () => patchMyReservationStatus(id, 'canceled'),
+    onSuccess: () => {
+      notify('success', '예약이 취소되었습니다.');
+      refetch();
```

Comment:
invalidateQueries를 쓰는 것도 좋을 것 같아요!

```suggestion
            queryClient.invalidateQueries({ queryKey: ['reservations'] });

```